### PR TITLE
Example notebook for Spectral generation and visualization

### LINKED ISSUE #130: Add notebook example for spectral generation and visualization

### PR DESCRIPTION
**Pull request by dzenanz**
Starting point was: https://github.com/thewtex/UltrasoundPhantomSegmentation/blob/6554d04ea1fb09aee883d1858e620a492a4c822c/distributions.ipynb

Closes #130.

### CODE REVIEW

**Comment by dzenanz** [comment]
@tbirdso please try it.

**Comment by tbirdso** [comment]
@dzenanz Thanks for putting this together! Would it be possible to add a line to install notebook dependencies as well as a brief descriptions of plots? I'm not sure I understand what the SingleGroup plots at the end of the notebook represent.

**Comment by thewtex** [comment]
Can we put a symlink for `examples/requirements.txt` -> `../.binder/requirements.txt`?

**Comment by thewtex** [comment]
@dzenanz can `zarr`, `xarray`, etc. be added to the *requirements.txt* file to fix the notebook CI?

**Comment by dzenanz** [comment]
Now we have:
```text
A cell timed out while it was being executed, after 300 seconds.
The message was: Cell execution timed out.
Here is a preview of the cell contents:
-------------------
# inclusion
drawSpectraCorrelations(inclusion_features.values, sample_size=1000)
-------------------
```

File: .binder/requirements.txt [hunk 1]
```diff
@@ -2,3 +2,8 @@ itk-ultrasound>=0.3.2
 itkwidgets
 matplotlib
 requests
+zarr
+xarray
+pandas
+seaborn
+sklearn
```
Comment (dzenanz) [inline .binder/requirements.txt:9]: I left out numpy, counting that https://github.com/InsightSoftwareConsortium/ITKPythonPackage/pull/159 will get merged eventually, and most people already have numpy anyway.

**Comment by dzenanz** [comment]
I beautified history and force-pushed. CI was green. Ready for another reivew/merge.

**Review by thewtex — APPROVED** [review]

**Pull request merged**